### ISSUE TITLE
Icons for each file type 

### ISSUE THREAD

**Issue by Loreehh** [face-hh/griddycode#104]
It would be nice if you could put a different icon (the logo of the programming language)  for each individual different file type, for example an icon for html, one for Python etc

**Comment by face-hh** [comment]
It would be entirely possible, but I don't think it's worth the time investment to create it.

If you wish, you can download the source code, download Godot, and modify the "icons.gd" file with another Nerdfont unicode :)

**Comment by Loreehh** [comment]
ok, thank you!

**Comment by not-yuvi** [comment]
> It would be nice if you could put a different icon (the logo of the programming language) for each individual different file type, for example an icon for html, one for Python etc


> It would be entirely possible, but I don't think it's worth the time investment to create it.
> 
> If you wish, you can download the source code, download Godot, and modify the "icons.gd" file with another Nerdfont unicode :)

IMO embedding https://github.com/PKief/vscode-material-icon-theme directly into bgc would be a great choice for icons if revisited in the future

**Comment by Loreehh** [comment]
> > It would be nice if you could put a different icon (the logo of the programming language) for each individual different file type, for example an icon for html, one for Python etc
> 
> 
> > It would be entirely possible, but I don't think it's worth the time investment to create it.
> > 
> > If you wish, you can download the source code, download Godot, and modify the "icons.gd" file with another Nerdfont unicode :)
> 
> IMO embedding https://github.com/PKief/vscode-material-icon-theme directly into bgc would be a great choice for icons if revisited in the future

yeah, it would be really cool if icons were added from that extension. I'm so clean
